content/material/2019-12-13-intro-rmarkdown.Rmd

```diff
@@ -33,10 +34,10 @@ install.packages("rmarkdown")
 ### Objectives
 
 1. To become aware of the importance of reproducibility of data analyses.
-1. To learn how to generate high quality reports that can be shared with a broader audience.
-1. To create reproducible documents interwoven with R code that can be run on updated or different data sets.
-1. To use Rstudio not as a data science tool but also as a text editor and compiler!
-1. To know where to go for continued learning.
+2. To learn how to generate high quality reports that can be shared with a broader audience.
```

Comment:
Numbering of these items is done automatically with rmarkdown, so they can be all listed as "1.". Just FYI, you don't need to change it now.